doc/api/buffer.md

```diff
@@ -5281,10 +5281,12 @@ An alias for [`buffer.constants.MAX_STRING_LENGTH`][].
 
 <!-- YAML
 added: v16.7.0
+changes:
+ - version: REPLACEME
+   pr-url: https://github.com/nodejs/node/pull/57513
+   description: Marking the API stable.
 -->
 
```

Comment:
```suggestion

> Stability: 2 - Stable

```

Reply:
That would be redundant with https://github.com/nodejs/node/blob/e4f2c25527a953ba524058cbb15fd1db60a820e1/doc/api/buffer.md#L5

Reply:
I disagree but dont care enough so whatever